docs/docs/dev-setup.md

```diff
@@ -4,9 +4,189 @@ title: Basic Setup
 sidebar_label: Basic Setup
 ---
 
-1. Install `west` from pip.
-1. Install the ARM SDK from Zephyr project.
-1. (Optional) Install 32-bit libs for building and testing locally.
-1. `west init -l zmk`
-1. Build it.
-1. Flash it.
+## Prerequisites
+
+A unix-like environment with the following base packages installed:
+
+- Git
+- Python 3
+- `pip`
+- `wget`
+- devicetree compiler
+- CMake
+- `dfu-util`
+- Various build essentials, e.g. gcc, automake, autoconf
+
+### Debian/Ubuntu
+
+On Debian and Ubuntu, this can be accomplished with:
+
+```bash
+apt-get install -y \
```

Comment:
On a fresh install you need to do apt update

```suggestion
sudo apt update
sudo apt install -y \
```

sudo and apt vs apt-get is just personal preference. However you want it works.